vNext/extensions/applicationinsights-dependencies-js/src/ajaxRecord.ts

```diff
@@ -57,10 +57,12 @@ export class ajaxRecord {
 
 
     public id: string;
+    public spanId: string;
 
-    constructor(id: string, logger: IDiagnosticLogger) {
+    constructor(id: string, logger: IDiagnosticLogger, spanId?: string) {
```

Comment:
This constructor signature looks ambiguous. I think instead a separate static `.fromW3CId(...)` can be made which returns a new `ajaxRecord` instance.